docs/adrs/0015-webapps-qa-strategy.md

```diff
@@ -0,0 +1,72 @@
+# Implementing QA-Only Mode for Web Apps to Test Features
+
+* **Status**: Accepted
+* **Deciders**: Yashika Khurana, Jared Lockhart, Benjamin Forehand
+* **Date**: 2024-06-14
+
+## Context and Problem Statement
+We have identified a significant gap in our QA testing process. Bugs are often not detected until features are live in production, despite thorough testing and QA in staging environments. This issue arises from differences in data and user interaction patterns between staging and production environments. To address this, we require a method to test features in production without affecting real users.
```

Comment:
We have identified a significant gap in our QA testing process *for webapps that integrate with Cirrus